src/olympia/abuse/views.py

```diff
@@ -208,31 +208,32 @@ def filter_enforcement_actions(enforcement_actions, cinder_job):
 def process_webhook_payload_decision(payload):
     source = payload.get('source', {})
     log.info('Valid Payload from AMO queue: %s', payload)
-    if 'job' in source:
+    if source.get('type') == 'api_decision':
```

Comment:
Looking at the payload I got from Cinder, `type` should be under `decision` in `source`, not `source` itself. Otherwise following your STR I'm getting a 400 Error with `Unsupported Manual decision`.

Reply:
😬 corrected, and included a (sanitized) real payload to test against https://github.com/mozilla/addons-server/pull/23386/commits/8635168147d24be9decf8abfd777f164f0a930ae